examples/with-vue-thirdpartyemailpassword/src/views/AuthView.vue

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import SuperTokensReactComponent from "../components/Supertokens";
+
+export default {
+
+    mounted() {
+        this.render();
+    },
+    beforeDestroy() {
+        ReactDOM.unmountComponentAtNode(document.getElementById("app") as Element);
+    },
+    methods: {
+        render() {
+            ReactDOM.render(React.createElement(SuperTokensReactComponent), document.getElementById("app"));
+        },
+    },
```

Comment:
You can remove the render function and just call `ReactDOM.render(React.createElement(SuperTokensReactComponent), document.getElementById("app"));` in `mounted`